client/src/components/Fieldset.js

```diff
@@ -33,7 +33,7 @@ const Fieldset = ({
       id={id}
       name={id}
       className="scroll-anchor-container"
-      style={style}
+      style={{ style, marginBottom: title === 'Cancelled Reports' ? '300px' : '' }}
```

Comment:
You should probably use `...style` here?

Reply:
Also, the bottom margin for any fieldset is 4rem, but `index.css` makes an exception for (a.o.) `#cancelled-reports fieldset` because we don't want to waste that space at the bottom of a page. So in short, this style override is probably not the way you should do it, and it also doesn't actually solve the issue.